wear/src/main/kotlin/io/homeassistant/companion/android/tiles/OpenTileSettingsActivity.kt

```diff
@@ -17,7 +17,8 @@ class OpenTileSettingsActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        val tileId = intent.extras?.getInt("com.google.android.clockwork.EXTRA_PROVIDER_CONFIG_TILE_ID")
+//        val tileId = intent.extras?.getInt("com.google.android.clockwork.EXTRA_PROVIDER_CONFIG_TILE_ID")
+        val tileId = intent.extras?.getInt("tile_id")
```

Comment:
I don't have the history here maybe @dshokouhi or @jpelgrom can help. But it looks like the extra key is not used at all. I'm ok about changing the key but you need to follow the guideline of the project and extract this into a const to avoid any typo.

I would prefer to see something like this https://github.com/home-assistant/android/blob/f2a31948b481760dbcb33137de42bf09ef9a5e14/app/src/main/kotlin/io/homeassistant/companion/android/settings/SettingsActivity.kt#L51 where we are not working with string directly and use strong types instead.

Reply:
The commented out tile ID is used when editing the tile from the carrousel on the watch (press and hold on the added tile & tap Edit). This functionality is now broken.

You should check for both extras to get the tile ID.

Reply:
@Martreides you should check this